examples/gno.land/p/gov/dao/members.gno

```diff
@@ -0,0 +1,34 @@
+package dao
+
+import (
+	"std"
+)
+
+// MemberStore defines the member storage abstraction
+type MemberStore interface {
+	// GetMembers returns all members in the store
+	GetMembers() []Member
```

Comment:
i think that we should not return a slice but have a GetByID(), + know the SIze()

or support pagination

Reply:
Added Size() as part of another suggestion

Added pagination:


e1cd617